dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/ProcessInstance.java

```diff
@@ -202,7 +202,7 @@
      * @return
      */
     @TableField(exist = false)
-    private Long duration;
+    private String duration;
```

Comment:
hi, should we separate this and create different beans?
It is not recommended to use one for DO, DTO, and VO.